content/zh/docs/concepts/context-propagation.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 上下文传播
+weight: 10
+description: 了解实现分布式追踪的概念。
+---
+
+通过上下文传播，[信号](/docs/concepts/signals)可以相互关联，
+无论它们是在何处生成的。
+尽管它不仅限于链路追踪，但它允许 [trace](/docs/concepts/signals/traces)
```

Comment:
It seems we have different translations for the same word, would be good to keep consistency. https://github.com/open-telemetry/opentelemetry.io/blob/43fea35b57456a18a0ef4b78072819f76a1c4fdd/content/zh/docs/what-is-opentelemetry.md?plain=1#L9

Reply:
Is this change blocking @reyang?

Reply:
Not blocking, we can cover it in separate PRs.

Reply:
Ok TY, merging!

Reply:
Issue:

- #4603